llama_stack/core/library_client.py

```diff
@@ -238,7 +234,24 @@ def __init__(
         self.provider_data = provider_data
         self.route_impls: RouteImpls | None = None  # Initialize to None to prevent AttributeError
 
+    def _remove_root_logger_handlers(self):
+        """
+        Remove all handlers from the root logger. Needed to avoid polluting the console with logs.
+        """
+        root_logger = logging.getLogger()
+
+        for handler in root_logger.handlers[:]:
+            root_logger.removeHandler(handler)
+            logger.info(f"Removed handler {handler.__class__.__name__} from root logger")
+
     async def initialize(self) -> bool:
+        """
+        Initialize the async client. Can be called multiple times safely.
```

Comment:
I don't think the second sentence is true. it cannot be called multiple times.

Reply:
fixed 👍🏽